test/jdk/sun/security/ssl/CipherSuite/AbstractDisableCipherSuites.java

```diff
@@ -0,0 +1,295 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
We should include 2018 in this copyright since most of it was copied from `NoDesRC4DesEdeCiphSuite.java`, so:

`* Copyright (c) 2018, 2024, Oracle and/or its affiliates. All rights reserved.`